### PR TITLE
Add run ECS task script

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: docker/aws/run_task.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+stage = sys.argv[2]
+cluster = sys.argv[3]
+task = sys.argv[4]
+
+boto_session = get_session(account_number, "TDRJenkinsLambdaRole" + stage.capitalize())
+
+client = boto_session.client('ecs')
+ec2_client = boto_session.client("ec2")
+
+security_groups = [x['GroupId'] for x in list(filter(lambda x: x['GroupName'] == "tdr-outbound-only", ec2_client.describe_security_groups()['SecurityGroups']))]
```
Comment (TomJKing) [inline docker/aws/run_task.py:15]: Maybe use something a bit more descriptive than x  for the variable?

Comment (MancunianSam) [reply docker/aws/run_task.py]: It's a fair point. I've changed both places where it's x

Comment (TomJKing) [reply docker/aws/run_task.py]: Still seeing: `... list(filter(lambda x: x['GroupName'] ...`?

Comment (MancunianSam) [reply docker/aws/run_task.py]: Ah, missed that one. I've put it in another variable to make it a bit easier to read and changed this x.

**Pull request merged**